### ISSUE TITLE
Not shuffling training data in updated zipformer receipt

### ISSUE THREAD

**Issue by mohsen-goodarzi** [k2-fsa/icefall#1137]
I noticed that in the updated zipformer receipt, the shuffled cuts are not used! Is it on purpose?

https://github.com/k2-fsa/icefall/blob/d667dc365b3259179c9d54bd32a1bb2bd8afa4f0/egs/librispeech/ASR/zipformer/train.py#L1179

**Comment by danpovey** [comment]
Ah, so you are wondering why it does:
```
train_cuts = librispeech.train_clean_100_cuts()
    if params.full_libri:
        train_cuts += librispeech.train_clean_360_cuts()
        train_cuts += librispeech.train_other_500_cuts()
```
and not:
```
    if params.full_libri:
         train_cuts = librispeech.train_all_shuf_cuts()
```
... yes, I wonder myself.  It's possible that we tried and found that the shuffled cuts didn't improve results, but I'm not sure.
Perhaps we just decided not to change the recipe for back-compatibility reasons or comparison of experiments, or not to force the user to shuffle the cuts on disk.  Hopefully one of the guys can comment.

**Comment by csukuangfj** [comment]
> Perhaps we just decided not to change the recipe for back-compatibility reasons or comparison of experiments

Yes, I think you are right.

The changes are introduced in #574 and #869.

Maybe @yaozengwei has forgotten the above two PRs so shuffled data is not used in the latest zipformer.

**Comment by mohsen-goodarzi** [comment]
One more thing:
Here #574,  it is shown that shuffling training cuts in prepare.sh is beneficial for reducing WER, and smother training loss curves, but isn't shuffling already enabled via sampler?

https://github.com/k2-fsa/icefall/blob/74806b744b81620d06645c27f5a2dda307e58322/egs/librispeech/ASR/tdnn_lstm_ctc/asr_datamodule.py#L157C20-L157C20
